tests/test_connection_options.py

```diff
@@ -30,25 +30,25 @@ async def test_postgres_pool_size_connect():
 
 
 def test_postgres_explicit_pool_size():
-    backend = PostgresBackend("postgres://localhost/database", min_size=1, max_size=20)
+    backend = PostgresBackend("postgres+psycopg://localhost/database", min_size=1, max_size=20)
```

Comment:
this was correct because it was testing for the driver without async aschema as well

Reply:
psycopg handles both ;)

It is simply an upgrade to psycopg3

Reply:
Odd, I was sure I had that, anyway good catch. We have the CI failing but will be good to add a develop branch from the main and open against that?